build.sbt

```diff
@@ -108,6 +108,8 @@ lazy val runtime = CrossProject(
   )
   .jsSettings(
     scalaJSStage := FastOptStage
+    // Uncomment to test JSDOM
+    // jsEnv := new org.scalajs.jsenv.jsdomnodejs.JSDOMNodeJSEnv()
```

Comment:
I was lazy to set this up in CI, sorry 🙃

Reply:
Also, this requires JSDOM to be installed via npm.
https://www.npmjs.com/package/jsdom